hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/infoschema/MappingColumnsTable.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.infoschema;
+
+import com.hazelcast.jet.sql.impl.schema.MappingDefinition;
+import com.hazelcast.jet.sql.impl.type.QueryDataTypeUtils;
+import com.hazelcast.sql.impl.schema.ConstantTableStatistics;
+import com.hazelcast.sql.impl.schema.TableField;
+import com.hazelcast.sql.impl.type.QueryDataType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+/**
+ * Table object for the {@code information_schema.columns} table.
+ */
+public class MappingColumnsTable extends InfoSchemaTable {
+
+    private static final String NAME = "columns";
```

Comment:
AFAIK `columns` is a reserved name for the view that exposes table columns. Given that we will need `inforamtion_schema` in IMDG in the future, I would propose to rename this view to `mapping_columns`. Otherwise, we will violate the standard.

Reply:
In `information_schema` the `columns` table is used columns of both tables and views. I think it can also be used for columns of `mappings`. I think when you're doing it, we need to integrate our work and display all columns there.

Reply:
True, but `columns` is part of the standard, and it must have columns with specific names: `table_name` instead of `mapping_name`, etc (see "Schemata" document of the standard, the paragraph "COLUMNS view"). The current implementation violates the standard. We should either rename columns or rename the table.